libraries/AP_HAL_ChibiOS/hwdef/mini-pix/hwdef.dat

```diff
@@ -148,3 +148,5 @@ define HAL_GPIO_C_LED_PIN 2
 define HAL_GPIO_LED_ON  0
 define HAL_GPIO_LED_OFF 1
 
+include ../include/minimize_common.inc
```

Comment:
Again, shouldn't be used directly; probably use `minimize_features.inc`?

Reply:
we decided to go with fpv_osd to get the rx telem an hd goggles